src/MainWindow.vala

```diff
@@ -1317,5 +1323,34 @@ namespace PantheonTerminal {
         public GLib.SimpleAction? get_simple_action (string action) {
             return actions.lookup_action (action) as GLib.SimpleAction;
         }
+
+        private class AccelMenuLabel : Gtk.Grid {
```

Comment:
I wonder how much of this class could be abstracted into Granite?  However, that is for the future.

Reply:
I have a PR for that :) https://github.com/elementary/granite/pull/300